tests/test_queuing.py

```diff
@@ -162,145 +191,47 @@ def test_append_qubit_observables(self):
 
     def test_append_tensor_ops(self):
         """Test that ops which are used as inputs to `Tensor`
-        are successfully added to the queue, but no `Tensor` object is."""
+        are successfully added to the queue, as well as the `Tensor` object."""
 
-        with qml._queuing.Queue() as q:
+        with Queue() as q:
             A = qml.PauliZ(0)
             B = qml.PauliY(1)
             tensor_op = qml.operation.Tensor(A, B)
-        assert q.queue == [A, B]
+        assert q.queue == [A, B, tensor_op]
```

Comment:
Don't we annotate `Tensor` objects anymore? 🤔 How come they are added to the queue?

Reply:
Oh this was a bug! The annotation still happens, but the Tensor operation _should_ have been added to the queue alongside the annotation.

This was fixed last year in tape mode, and here we are just updating the 'old' test for this 'new' behaviour.